backend/import/mo_api.py

```diff
@@ -11,10 +11,11 @@
 from collections import defaultdict
 
 import requests
-global_session = requests.session()
 
 from cached_property import cached_property
 
+global_session = requests.session()
```

Comment:
Apparently, `requests.session` is “for context-management”. The proper name of the class is:

```suggestion
global_session = requests.Session()
```